pkg/yurtmanager/webhook/node/v1/node_default.go

```diff
@@ -38,9 +38,14 @@ func (webhook *NodeHandler) Default(ctx context.Context, obj runtime.Object, req
 		return apierrors.NewBadRequest(fmt.Sprintf("expected a Node but got a %T", obj))
 	}
 
-	npName, ok := node.Labels[apps.NodePoolLabel]
-	if !ok {
-		return nil
+	npName := node.Labels[apps.NodePoolLabel]
+	if len(npName) == 0 {
+		npName = node.Labels["apps.openyurt.io/desired-nodepool"]
```

Comment:
Why isn't this label maintained as a constant, just like `apps.NodePoolLabel`

Reply:
fixed